src/heretic/g_game.c

```diff
@@ -1079,6 +1084,11 @@ static boolean InventoryMoveRight()
 
     plr = &players[consoleplayer];
     inventoryTics = 5 * 35;
+
+    if (MenuActive)
```

Comment:
Applies to both Heretic and Hexen, not sure if this is optimal, but we have to prevent inventory pop-up while using mouse wheel in active menu.